src/Hero.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Random;
+
+public class Hero {
```

Comment:
jedna rzecz o której bym się zastanowił to ile wspólnego (co wspólnego?) ma bohater i kamień i ewentualne nowe rzeczy które będziemy mogli dodawać. Może da się wyłuskać tę część do jakiejś klasy bazowej? Ale to już w następnym Pull requeście